app/assets/stylesheets/utilities/all.scss

```diff
@@ -0,0 +1,8 @@
+@import 'background';
```

Comment:
QQ: why didn't we just leave these in`components/all.scss`?

Reply:
I wasn't initially expecting to need to make this change. In terms of precedence, we want utilities to be stronger than components, and we want our custom components and utilities to be stronger than the design system. Without this change, the design system utilities were overriding our custom utility extensions. I noticed this caused some issue in the step indicator, where [our custom margin classes](https://github.com/18F/identity-idp/blob/573fae5b8a2a9f30383b32e9953c29cafbde52c1/app/assets/stylesheets/components/_space-addon.scss#L1-L9) were being overwritten by [unprefixed (mobile-first) base margin classes](https://github.com/18F/identity-idp/blob/573fae5b8a2a9f30383b32e9953c29cafbde52c1/app/views/layouts/flow_step.html.erb#L8).

![image](https://user-images.githubusercontent.com/1779930/116702171-bf9f2a80-a996-11eb-8e82-a774129d6194.png)

I'm hoping we can gradually remove a lot of these utilities, since many of them can and should be accomplished using design system equivalents. That said, I do find this to be a bit of a clearer separation of styles, which is nice.